ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml

```diff
@@ -13,6 +13,7 @@
       - docker_cluster_collocation
       - purge_bluestore_osds_container
       - purge_bluestore_osds_non_container
+      - ooo_collocation
```

Comment:
Do we really want this to start on every PR? Fine if so, but I was initially thinking it'd be something that we'd request manually.

Reply:
according to @fultonj yes, it's expected to be run on every PR